src/services/user.service.js

```diff
@@ -0,0 +1,33 @@
+const { User } = require('../models/User.model.js');
+
+const getAllUsers = async () => {
+  return User.findAll();
+};
+
+const getUserById = async (id) => {
+  return User.findByPk(id);
+};
+
+const createUser = async (name) => {
+  return User.create({ name });
```

Comment:
Consider adding validation or error handling to ensure that the name being created is valid and to handle any potential errors that might occur during the creation process.